pkg/oauthflow/flow.go

```diff
@@ -94,7 +94,7 @@ var PublicInstanceMicrosoftIDTokenGetter = &InteractiveIDTokenGetter{
 }
 
 // OIDConnect requests an OIDC Identity Token from the specified issuer using the specified client credentials and TokenGetter
```

Comment:
can you add a comment for this function denoting the use of redirectURI (if blank, you get `localhost:0` but otherwise you will get an HTTP listener at the hostname:port specified)?

Reply:
Done :).